core/tools/scripts/bootstrap.sh

```diff
@@ -3,5 +3,6 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
-SCRIPTS_PATH=$(dirname "$0")
-cp $PWD/$SCRIPTS_PATH/red-pill /usr/bin/
+LOCAL_BINARY_PATH="/usr/bin/red-pill"
+
+cp /home/pi/tools/scripts/red-pill $LOCAL_BINARY_PATH
```

Comment:
You can rely that the source will be in this path, it's necessary to use dirname for relative access. Can you give an example where this fail ?

Reply:
I have set `WORKDIR=/home/pi` on the BlueOS-base's Dockerfile, then the old method expands as `cp /home/pi//home/pi/tools/scripts/red-pill /usr/bin/pi`, obviously wrong.

The `red-pill` will be at `/home/pi/tools/scripts/` because of `COPY tools /home/pi/tools` ([from core's Dockerfile](https://github.com/bluerobotics/BlueOS-docker/blob/master/core/Dockerfile#L19)).

Reply:
It could also be:
```suggestion
cd /
cp $PWD/$SCRIPTS_PATH/red-pill /usr/bin/
cd $OLDPWD
```

Reply:
yeah, $(dirname $(realpath $0)) does, but [this](https://stackoverflow.com/a/246128) looks better.